src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/CachedBitmap.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Gdip = System.Drawing.SafeNativeMethods.Gdip;
+
+namespace System.Drawing.Imaging
+{
+    /// <summary>
+    /// Stores a <see cref="Bitmap"/> in a format that is optimized for display on a particular device.
+    /// </summary>
+    public sealed partial class CachedBitmap : IDisposable
+    {
+        internal static bool IsSupported = IsCachedBitmapSupported();
+        internal IntPtr nativeCachedBitmap;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CachedBitmap"/> class.
+        /// </summary>
+        /// <param name="bitmap">The bitmap to take the pixel data from.</param>
+        /// <param name="graphics">A <see cref="Graphics"/> object, representing the display device to optimize the bitmap for.</param>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="bitmap"/> is <see langword="null" />.
+        /// - or -
+        /// <paramref name="graphics"/> is <see langword="null" />
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
+        /// The installed version of libgdiplus does not support CachedBitmap. This does not apply on Windows.
```

Comment:
Hint for 6.1 or higher?
As a consumer I'd like to see how to fix it in a easy way 😉